implants/eldritch/src/pivot/port_scan_impl.rs

```diff
@@ -529,8 +502,7 @@ mod tests {
         let expected_response: Vec<(String, i32, String, String)>;
         expected_response = vec![(host.clone(),test_ports[0],proto.clone(),"open".to_string()),
                 (host.clone(),test_ports[1],proto.clone(),"open".to_string()),
-                (host.clone(),test_ports[2],proto.clone(),"open".to_string()),
-                (host.clone(),test_ports[3],proto.clone(),"closed".to_string())];
+                (host.clone(),test_ports[2],proto.clone(),"open".to_string())];
```

Comment:
Code would be more readable if we defined "open" as a constant or a variable, a lot of extra code calling `to_string()` inline makes it a bit more difficult to read this

Reply:
We'll still have to call `to_string()` since const can only be a &str.
Updated to consts though.